lib/pbench/cli/server/user_create.py

```diff
@@ -30,8 +31,14 @@ def execute(self):
             last_name=self.context.last_name,
             email=self.context.email,
         )
+        if self.context.role:
+            user.role = self.context.role
```

Comment:
Since `role` is an attribute/column of `User`, shouldn't it be supplied to the c'tor?  It _is_ nullable, so I suppose it's a question of style, but something like this might be appropriate:
```
        user = User(
            username=self.context.username,
            password=self.context.password,
            first_name=self.context.first_name,
            last_name=self.context.last_name,
            email=self.context.email,
            role=self.context.role if self.context.role else None,
        )
```
I am prone to over-using the "ternary" (or conditional) operator, but I do like having all of the attributes provided at construction -- it tends to help keep them together.